packages/build-info/tests/detect-build-systems.test.ts

```diff
@@ -100,3 +100,14 @@ test('detects moonrepo when .moon directory is present', async () => {
   const buildSystems = await detectBuildSystems(cwd)
   expect(buildSystems[0]).toEqual({ name: 'moon', version: '^0.5.1' })
 })
+
+test('detects build system in a monorepo setup', async () => {
+  const cwd = mockFileSystem({
+    'packages/website/package.json': JSON.stringify({ devDependencies: { turbo: '^1.6.3' } }),
+    'packages/website/turbo.json': '',
+    'packages/server/server.js': '',
+  })
+
+  const buildSystems = await detectBuildSystems('packages/website', cwd)
```

Comment:
I think the values that are passed here are absolute paths 🤔

Reply:
It doesn't make a difference because `getContext` resolves the absolute paths.

https://github.com/netlify/build/blob/34b80c6c3840735284a3907f5898f0e4ffe2fa52/packages/build-info/src/context.ts#L36-L44